frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeContent/SummarizeContent.tsx

```diff
@@ -0,0 +1,82 @@
+import { t } from "ttag";
+
+import * as Lib from "metabase-lib";
+
+import { AddAggregationButton } from "../AddAggregationButton";
+import { AggregationItem } from "../AggregationItem";
+import { BreakoutColumnList } from "../BreakoutColumnList";
+import {
+  AggregationsContainer,
+  ColumnListContainer,
+  SectionTitle,
+} from "../SummarizeSidebar.styled";
+
+import { STAGE_INDEX } from "./use-summarize-query";
+
+export type SummarizeContentProps = {
+  query: Lib.Query;
+  aggregations: Lib.AggregationClause[];
+  hasAggregations: boolean;
+  onAddAggregation: (aggregations: Lib.Aggregable) => void;
```

Comment:
Why is this different than the original PR where it's `onAddAggregations`?

Reply:
I had the same question. It looks like the `onAddAggregations` functionality was never added, and I was getting quite a few errors down the component tree.

Reply:
Let me know how to proceed if it's an issue here

Reply:
Based on our conversation, [**#43220 Previous period comparison shortcut in notebook mode**](https://github.com/metabase/metabase/pull/43220) not being backported will let us backport without much concern :crossed\_fingers: